wallet/src/account/output_cache/mod.rs

```diff
@@ -239,11 +239,17 @@ impl OutputCache {
             .collect()
     }
 
-    pub fn abandon_transaction(&mut self, tx_id: Id<Transaction>) -> WalletResult<()> {
+    pub fn abandon_transaction(
+        &mut self,
+        tx_id: Id<Transaction>,
+    ) -> WalletResult<Vec<Id<Transaction>>> {
```

Comment:
Please add docs explaining what this function returns

Reply:
fixed